lms/djangoapps/teams/static/teams/js/collections/topic.js

```diff
@@ -1,4 +1,4 @@
-;(function (define) {
+(function(define) {
```

Comment:
Yeah, @bjacobel, it's that we almost got stung last week by a big pylint fixing PR in which there were autocorrect changes that would have changed the meaning of the code, so I'm sensitive to it this week. Only a read-through that caught it.

Agree on brainstorming through a plan if we do another huge scale change. We might make commits per-violation-pattern or something to break it down.